src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/Vilkår/BarnetsAlder/BarnetsAlder.tsx

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 
+import { isBefore } from 'date-fns';
+
 import { Label, Radio, RadioGroup } from '@navikt/ds-react';
 
 import { muligeUtdypendeVilkårsvurderinger, useBarnetsAlder } from './BarnetsAlderContext';
-import { Resultat, VilkårRegelsett } from '../../../../../../typer/vilkår';
+import { Resultat } from '../../../../../../typer/vilkår';
+import { parseTilOgMedDato, type IIsoDatoPeriode } from '../../../../../../utils/dato';
 import type { IVilkårSkjemaBaseProps } from '../../VilkårSkjema';
 import { VilkårSkjema } from '../../VilkårSkjema';
 import { useVilkårSkjema } from '../../VilkårSkjemaContext';
 
 type BarnetsAlderProps = IVilkårSkjemaBaseProps;
 
-const hentSpørsmålBasertPåRegelsett = (regelsett: VilkårRegelsett) => {
-    switch (regelsett) {
-        case VilkårRegelsett.LOV_AUGUST_2021:
+const FØRSTE_FOMDATO_LOV_AUGUST_2024 = new Date('2024-08-01T00:00:00+02:00');
```

Comment:
Det finnes en variabel for dette i dato.ts - filen :)

Reply:
Nice! Sjekket ikke